.github/ISSUE_TEMPLATE/1-leak.md

```diff
@@ -13,5 +13,5 @@ This issue tracker is not for help with memory leaks detected by LeakCanary in y
 
 To fix a leak: 
 
-* First, learn the fundamentals: https://github.com/square/leakcanary#fundamentals
+* First, learn the [fundamentals](/docs/fundamentals.md)
```

Comment:
The proper link should be `https://square.github.io/leakcanary/fundamentals/` not `/docs/fundamentals.md`

Reply:
Done